packages/wallet-sdk/src/util/validatePreferences.ts

```diff
@@ -9,6 +9,15 @@ export function validatePreferences(preference?: Preference) {
     return;
   }
 
+  if (
+    preference.keysUrl &&
+    !['https://keys.coinbase.com/connect', 'https://keys-dev.coinbase.com/connect'].includes(
+      preference.keysUrl
+    )
+  ) {
+    throw new Error(`Invalid keysUrl: ${preference.keysUrl}`);
+  }
+
```

Comment:
Should we validate this?

Reply:
Might not be a bad idea. The config has the field listed as deprecated and for internal use only.